content/pt/docs/referencia-da-api/pix/apis-stone/reivindicacoes/listar-reivindicacoes/_index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Listagem de Reivindicação/Portabilidade"
+linkTitle: "Listagem de Reivindicação/Portabilidade"
+date: 2022-09-17T18:00:00-03:00
+lastmod: 2022-11-23T11:00:00-03:00
+weight: 10
+draft: true
+description: >
+
+---
+
+##### **Request**
+---
+
+```
+GET /api/v1/pix/:account_id/entry_claims
+```
+
+##### **Paramentros para query**
+---
+```
+"status"
+"key"
+"direction"
+"limit"
+"after"
+"before" 
+```
+
+<br>
+
+**status** `array` `string` 
+<br>
+Valores permitidos:  `waiting_resolution`, `open`, `cancelled`, `completed` ou `confirmed`
+
+<br>
+
+**direction** `string`
+<br>
+Valores permitidos: `outbound`, `inbound` 
+
+
+Exemplo:  
+
+- `GET -/api/v1/pix/:account_id/entry_claim?key[]=35425517084&key[]=93744525023`;
+<br> <br> 
+
+
```

Comment:
```suggestion
```

Reply:
@brvnaleao why? is it an unwritten rule for this kind of requests?